src/components/select/Select.tsx

```diff
@@ -43,6 +51,17 @@ export const Select = ({
   const { border300, radius100 } = borders;
   const { primaryB, primary100 } = colors;
 
+  const handleOnChange = (params: CustomParams) => {
+    if (onChangeHandler) {
+      if (params.value.length === 1) {
+        console.log(params.value);
```

Comment:
@Ihor-Lavrov is this console log necessary?

Reply:
Also maybe it's better to make it one-line like `value: params.value.length === 1 ? params.value[0] : params.value`?

Reply:
make sense, changed